ddprof-ffi/src/lib.rs

```diff
@@ -195,10 +195,10 @@ impl<'a> TryFrom<Slice<'a, u8>> for &'a str {
     }
 }
 
-impl<'a> TryFrom<Slice<'a, c_char>> for &'a str {
```

Comment:
Here is the error I was facing 

error[E0119]: conflicting implementations of trait `std::convert::TryFrom<Slice<'_, u8>>` for type `&str`:
   --> ddprof-ffi/src/lib.rs:198:1
    |
189 | impl<'a> TryFrom<Slice<'a, u8>> for &'a str {
    | ------------------------------------------- first implementation here
...
198 | impl<'a> TryFrom<Slice<'a, c_char>> for &'a str {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `&str`

Reply:
You read my mind.  I think these two tactical change are sufficient since ARM uses unsigned char.